core/templates/dev/head/pages/base.html

```diff
@@ -169,7 +169,12 @@ <h2>
     {% include 'pages/footer_js_libs.html' %}
 
     <script src="/templates/dev/head/app.js"></script>
-    <script src="/templates/dev/head/directives.js"></script>
+    <script src="/templates/dev/head/directives/AngularHtmlBindDirective.js"></script>
+    <script src="/templates/dev/head/directives/CustomPopoverDirective.js"></script>
+    <script src="/templates/dev/head/directives/FocusOnDirective.js"></script>
+    <script src="/templates/dev/head/directives/MathjaxBindDirective.js"></script>
+    <script src="/templates/dev/head/directives/MobileFriendlyTooltipDirective.js"></script>
+    <script src="/templates/dev/head/directives/SelectOnClickDirective.js"></script>
```

Comment:
We should load the scripts only when they're needed. So if they're not needed for _base.html_ to function move them only to pages where they're needed.

Reply:
Before making the changes, I just want to confirm that this means - I will check through pages in `core/templates/dev/head/pages` and include the scripts in the pages they are used in?

Reply:
Yes, exactly

Reply:
Ok, working on it.

Reply:
Done.